.dockerignore

```diff
@@ -0,0 +1,3 @@
+**/node_modules/
+**/build/
+**/.github/
```

Comment:
usually add .git/ is a good idea - because, we don't need this directory when we builds image

Reply:
Done